docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,139 @@
+# Running the Airy Core Plaform in production
+
+
+The cloud providers part can be left as TODO in the first pr. It's enough to cover all the requirements (size of a kubernetes cluster, how to get the code up and running in the cluster, how to run the services (redis, postgres, kafka, zookeeper) both in the same kubernets cluster or how to configure the apps so that the system works)
+
+## Architecture
+
+The Airy Core Platform is messaging platform, based on microservices and comprised of a backend and frontend system. 
+
+The `backend` system is a streaming platform and its role is to:
+- Receive messages or events from different sources, process them on the fly and store them in Kafka `(sources)`
+- Make them available and accessible through an API (api)
+- Send messages or events to external parties (webhooks)
```

Comment:
Here I would link to `glossary.md#sources`, `http.md` and `webhook.md`

Reply:
Do you mean instead of this text or as an addition?